conduit/src/Data/Conduit/Combinators.hs

```diff
@@ -1755,6 +1756,21 @@ mapAccumWhileC f =
 {-# INLINE mapAccumWhileC #-}
 STREAMING(mapAccumWhile, mapAccumWhileC, mapAccumWhileS, f s)
 
+
+-- | Specialized version of 'mapAccumWhile' that does not provide values downstream.
+--
+-- Subject to fusion
```

Comment:
I don't think this is true, since it isn't leveraging the STREAMING CPP. Can you remove it?